tsconfig.json

```diff
@@ -11,6 +11,8 @@
     "checkJs": false,
     // Disallow features that require cross-file information for emit as we're using Babel
     "isolatedModules": true,
+    "strict": true,
+    "resolveJsonModule": true,
```

Comment:
Some of the files import `package.json` and various JSON configs. This helps us avoid writing type definitions for them.

Reply:
Gotcha, wasn't too sure about adding this option before.